A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg

```diff
@@ -327,7 +327,7 @@ wingflex_scalar = 0.75
 wingflex_offset = -0.25
 
 [REFERENCE SPEEDS]
-full_flaps_stall_speed = 115 ; Knots True (KTAS)
+full_flaps_stall_speed = 121 ; Knots True (KTAS)
```

Comment:
why does this need to be changed?

Reply:
Oops, left some testing artifacts. Thanks for noticing. Also left one intended change unexplained. I will add an explanation for that one (pitch moment due to elevator movement).